jenkins-builds/leone-RHEL6.9EUS-17.06

```diff
@@ -0,0 +1,14 @@
+ Boost-1.61.0-foss-2016b-Python-2.7.12.eb
+ ddt-18.0.1.eb
+ ddt-18.1.1.eb                              --set-default-module
+ h-yade-2017.01a-foss-2016b-Python-2.7.12.eb
```

Comment:
The user is currently trying to rebuild yade on his own with EB (they need to re-compile it frequently and having a static version maybe is not so helpful).

For the moment we don't need to include this on the list, if the user does not succeed @ChristopherBignamini will ask to add it back to the list.